### ISSUE TITLE
AssistEdits on Option fields don't show correctly in Web Client

### ISSUE THREAD

**Issue by ShadowXVII** [Microsoft/AL#3786]
Assist Edits on Options fields do not show in the Web Client.
This prevents the Change Log from being configurable at the fields level.

This affects all versions (Next Major: 13.0.24284.0-W1, Current: 12.3.23139.0-W1) including existing tenants on Business Central (businesscentral.dynamics.com).

## Steps to repro
1. Open Change Log Setup
2. Open Table List
3. On any table, pick Some Fields

## Expected Result
User should be able to pick the fields using an assist edit button
![nav2018shownassistedit](https://user-images.githubusercontent.com/446329/45787424-97b49a00-bcb8-11e8-8172-d797568745f0.png)

## Actual Result
User cannot select Assist Edit button
![bcmastermissingassistedit](https://user-images.githubusercontent.com/446329/45787428-9a16f400-bcb8-11e8-87b3-5337aed31903.png)

This is a cross-post of the Collaborate bug (https://partner.microsoft.com/en-us/dashboard/collaborate/engagements/1173/feedback/wits/Bugs/131417)

**Comment by ShadowXVII** [comment]
It makes the Change Log "some fields" impossible to use.

That being said, it's one of the only places I can think of an option field with assist edit by standard.

**Comment by atoader** [comment]
What are the major difficulties you are encountering? Could you provide a more detailed account? I will forward your feedback to the client team and we will use it to improve the experience in future releases.

**Comment by ShadowXVII** [comment]
It's all in the screenshot -- in the Web Client it's not possible to setup because the Assist Edit button doesn't show. I'm not sure what more there is to tell.

How does a user get into the Assist Edit screen to pick the fields they want to track on the Change Log?

**Comment by ikoletic** [comment]
You're right. Similar pattern as for Dimension Combinations needs to be applied here. Tagged it as application enhancement. Will have team look at ti.

**Comment by ShadowXVII** [comment]
DImension Combinations is also affected by this, and therefore the functionality is rendered useless. This should be a bug 🐞, not an enhancement.

Either, the Assist Edit on options fields should be restored, or the UI should be revised to work around the limitation.

I'm still dumbfounded that this isn't picked up by Unit Tests.

## Windows Client
The AssistEdit can be used to access the options for the particular combination of dimensions.
![image](https://user-images.githubusercontent.com/446329/46322998-5f557a00-c62f-11e8-91ca-225e06389f5c.png)

## Web Client
The AssistEdit cannot be seen. Maybe this screen should use Drill-Downs instead? Though AssistEdit was more intuitive.
![image](https://user-images.githubusercontent.com/446329/46323001-62e90100-c62f-11e8-834e-8bbcdc542cc8.png)

**Comment by ShadowXVII** [comment]
For reference, this was still not fixed in Oct 18 release.

Still affected: 13.0 (25488)
Fixed in (potentially earlier): 14.0 (25812)
